terraform/variables.tf

```diff
@@ -12,24 +12,30 @@ variable "remote_state_bucket" {
 }
 
 variable "state_prefix" {}
+variable "workspace_key_prefix" {}
+variable "state_file_name" {}
 
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
-variable "vpc_cidr" {
-  default = {
-    eu-west-2 = "10.75.0.0/16" #London
-  }
+# Required for access from existing Concourse instances
+variable "dev_management_cidrs" {
+  type    = list(string)
+  default = ["10.50.17.0/24","10.50.19.0/24","10.50.21.0/24"]
 }
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
+
+# VPC ID read from terraform remote state.
 variable "vpc_id" {
-  default = {
-    eu-west-2 = "vpc-074ff55ed5182e144" #London
-  }
+  default     = ""
```

Comment:
Is there a reason we're specifying an empty default?

Reply:
Ah, OK. So I'd done this because otherwise it was prompting for a value. But on reflection, this is defined in one of those _controversial_ locals blocks - so doesn't need to be defined here at all. So removing definition entirely. 👍